frontend/app_flowy/packages/flowy_infra/lib/image.dart

```diff
@@ -1,15 +1,26 @@
 import 'package:flutter/widgets.dart';
 import 'package:flutter_svg/flutter_svg.dart';
 
-Widget svgWidget(String name, {Color? color}) {
-  final Widget svg = SvgPicture.asset('assets/images/$name.svg', color: color);
-
-  return svg;
-}
-
 Widget svgWithSize(String name, Size size) {
```

Comment:
this function is duplicated with the following function, I think we can remove this function.

Reply:
I will remove this function through a new PR. Because there are so many codes had referenced this function.